monetio/models/hysplit.py

```diff
@@ -199,7 +207,7 @@ def __init__(
 
         if readwrite == "r":
             if verbose:
-                print("reading " + filename)
+                logger.info(f"reading {filename}")
```

Comment:
This won't show unless user changes their logging config. Maybe such `verbose=True` message should stick with `print()`.